### PR TITLE
Consolidate variables and make file

### PR DESCRIPTION
**Pull request by marrobi**
# PR for issue #209

## What is being addressed

The repository had lots of duplicate variables both across Porter and Terraform, and across various components such as core and workspaces. These have been consolidated.

## How is this addressed

- Remove all TF_VAR_ .env files. Create script to convert names such as `ACR_NAME` to `TF_VAR_acr_name`
- Update porter parameter files to use consistant naming
- Update all scripts to use consistent naming
- Update docs to match 
- consolidate make file to use `DIR=` rather than having many independent steps.
- 
Passes CD pipeline here: https://github.com/microsoft/AzureTRE/actions/runs/936799251

### CODE REVIEW

**Review by sachinkundu — APPROVED** [review]
TL;DR
As long as it works and it does according to the pipeline.
